src/tools/config/getFrontendConfig.ts

```diff
@@ -7,6 +7,22 @@ export const getFrontendConfig = (name: string): ConfigType => {
   const config = getConfig(name);
 
   Consola.info(`Requested frontend content of configuration '${name}'`);
+  // Find out if config as apps with integrations that have a property with no value or undefined
+  // If so, remove print an error using consola
+  // If not, return the config
+  const appsWithIntegrationsWithUndefinedProperties = config.apps.filter(
+    (app) =>
+      app.integration?.properties.some(
+        (property) => property.value === null || property.value === undefined
+      ) ?? false
+  );
+  if (appsWithIntegrationsWithUndefinedProperties.length > 0) {
+    Consola.warn(
```

Comment:
Change to ``info``. I believe that we'll add more integration options in the future, that will not be required.

Reply:
imo it should be a warn. info is for stuff that does not break anything i.e. new version available or number of TV shows pulled bu calendar widget

Reply:
I think warn should be used for things, that cannot be handled by Homarr, but are not critical. But I'm fine with keeping this 👍